src/main/java/responseCodes/exception/BadRequestException.java

```diff
@@ -0,0 +1,8 @@
+package responseCodes.exception;
+
+public class BadRequestException extends Exception {
```

Comment:
Please compare Exception vs RuntimeException vs Throwable. What is best way in this case?